src/routes/Canvas.svelte

```diff
@@ -66,6 +71,12 @@
 						on:mousedown|stopPropagation={(e) => resizeStart(e, i)}
 						on:mouseup={resizeEnd}
 					/>
+					<!-- svelte-ignore a11y-click-events-have-key-events -->
+					<div 
+						class="remove"
+						style="position:absolute;top:-14px; right: -14px; overflow:visible"
```

Comment:
If you are adding a class, maybe style everything on the class

Reply:
Resolved